### PR TITLE
Make record in inventory on job completion

### PR DESCRIPTION
**Pull request by alexanderGerbik**
https://p.daedalean.ai/T6094

### CODE REVIEW

File: cvat/apps/engine/ddln/inventory_client.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import datetime as dt
+
+from django.conf import settings
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+
+from ..log import slogger
```
Comment (gyakushev) [inline cvat/apps/engine/ddln/inventory_client.py]: use full path instead

File: cvat/apps/engine/models.py [hunk 2]
```diff
@@ -193,6 +199,29 @@ class Job(models.Model):
...
+        _, sequence_name = parse_frame_name(image_path)
+        annotator = self.assignee.username if self.assignee else ''
+        annotation_date = date.today()
+
+        try:
+            client = create_inventory_client()
+            affected_cells = client.record_sequence_completion(sequence_name, task_name, annotator, annotation_date)
+            slogger.glob.info("Job %s completed. Made a record in inventory file: '%s'", self.id, affected_cells)
+        except Exception:
+            slogger.glob.exception("Error while making the job completion record")
```
Comment (gyakushev) [inline cvat/apps/engine/models.py:222]: print exception message

**Review by gyakushev — APPROVED** [review]
Great!

File: cvat/apps/engine/models.py (hunk 2 above)
Comment (alexanderGerbik) [reply cvat/apps/engine/models.py:222]: When `logger.expection()` is called within `except` block, the exception and the traceback are added automatically to the logs:
error

**Pull request merged**